packages/@react-aria/interactions/docs/interactions.mdx

```diff
@@ -109,9 +109,7 @@ so that components receive a consistent stream of events regardless of the inter
 All React Aria components are tested across a wide variety of browsers and devices. We test across devices with
 mouse input, touchscreens, and also hybrid devices.
 
-* Chrome 88+ on macOS and Windows
-* Firefox 85+ on macOS and Windows
-* Safari 13+ on macOS
-* Edge 88+ on Windows
-* Safari 13+ on iOS and iPadOS
-* Chrome 88+ on Android
+* Chrome on macOS, Windows, and Android
+* Firefox on macOS and Windows
+* Safari on macOS, iOS, and iPadOS
+* Edge on Windows
```

Comment:
These were really outdated. I don't think we have updated since the original release of RSP v3. Discussed with @dannify, and we think we can just omit the specific versions here.